src/components/button/Button.tsx

```diff
@@ -14,9 +14,14 @@ export type ButtonProps = {
   /** If true, substitutes the button content by an animated loader */
   loading?: boolean;
   type?: 'button' | 'reset' | 'submit';
-   /** The variant to use*/
-  variant?: 'primary' | 'secondary' | 'tertiary' | 'destructive';
   onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+   /** The variant to use*/
+  variant?: 'primary' | 'secondary' | 'tertiary';
+   /** Subvariant for each variant */
+  isDesctruvtive?: boolean;
+  /** Subvariant for destructive */
+  isActive?: boolean;
```

Comment:
"isAccent" I think

Reply:
Apart from that, I am wondering is adding two booleans isDestructive and isAccent is the right approach to handle the different variants...
@symphony-jean-michael @symphony-adnane

Reply:
What about variant= "primary-destructive", "tertiary-accent", etc? So we are clear about the variant we are handling

or keep separated like
```
variant: 'primary' | 'secondary' | 'tertiary'
subVariant: "destructive", "accent"
```